.github/workflows/avm.platform.sync-avm-modules-list.yml

```
@@ -0,0 +1,36 @@
+name: "avm.platform.sync-avm-modules-list"
```

```suggestion
name: .Platform - Sync AVM module list
```

in alignment with changes that will be introduced by #1411